### ISSUE TITLE
Instagram extension - a11y update

### ISSUE THREAD

**Issue by patHyatt** [ampproject/amphtml#3505]
Hey everyone,
Running an accessibility (using tota11y) check the amp-instagram component yields an issue for not containing an alt attribute value.

Secondly the iframe that is pulled in does not have a title attribute which can be problematic for screen readers. This is suggested for helping screen reader users differentiate frames.

I'm sitting on changes which allow for an alt attribute on the when using the amp-instagram extension, which will propagate that attribute to the underlying amp-img, and img elements. 

The alt text added will also set the pulled in iframe element's title attribute, prefixed by "Instagram:".

Would this be deemed worthy?

Example source:
```html
  <amp-instagram
      data-shortcode="6f8K16iZiK"
      alt="Second image"
      width="500"
      height="500"
      layout="responsive">
  </amp-instagram>
```

Example output from the examples.build/instagram.amp.max.html:
```html
<amp-instagram data-shortcode="6f8K16iZiK" alt="Second image" width="500" height="500" layout="responsive" class="-amp-element -amp-layout-responsive -amp-layout-size-defined -amp-layout" id="AMP_3">
   <i-amp-sizer style="display: block; padding-top: 100%;"></i-amp-sizer>
   <div placeholder="" class="amp-hidden">
      <amp-img class="-amp-element -amp-layout-fill -amp-layout-size-defined -amp-layout" src="https://www.instagram.com/p/6f8K16iZiK/media/?size=l" layout="fill" alt="Second image" id="AMP_4" style="top: 48px; bottom: 48px; left: 8px; right: 8px;"><img amp-img-id="AMP_4" alt="Second image" class="-amp-fill-content -amp-replaced-content" width="0" height="0" src="https://www.instagram.com/p/6f8K16iZiK/media/?size=l"></amp-img>
   </div>
   <iframe frameborder="0" allowtransparency="true" title="Instagram: Second image" src="https://www.instagram.com/p/6f8K16iZiK/embed/?v=4" class="-amp-fill-content" width="500" height="500" style="opacity: 1;"></iframe>
</amp-instagram>
```

**Comment by jridgewell** [comment]
I think that'd be great!

**Comment by patHyatt** [comment]
Believe this can be closed (correct?), PR #3520 has addressed it.

**Comment by jridgewell** [comment]
Whoops.
